.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+rc[0-9]+'
```

Comment:
possibly whatever kind of regex this is already does this but I guess we might need to escape the `.`s here?